Moroz_Homework_1.cpp

```diff
@@ -0,0 +1,59 @@
+#include <cmath>
+#include <iostream>
+#include <limits>
+
+constexpr uint16_t dividendMultiply(4);
+constexpr uint16_t dividerMultiply(2);
+
+constexpr uint16_t numberOfAttempts(3);
+
+int main() {
+  int16_t a(0);
```

Comment:
можно вместо явного задания литерала `0` использовать дефолтную инициализацию
```suggestion
  int16_t a{};
```